NuRadioMC/SignalGen/parametrizations.py

```diff
@@ -27,7 +27,7 @@ def set_log_level(level):
 
 def get_parametrizations():
     """ returns a list of all implemented parametrizations """
-    return ['ZHS1992', 'Alvarez2000', 'Alvarez2009', 'Alvarez2012']
```

Comment:
TLDR - because it doesn't exist

I just checked the file history - it looks like Alvarez2012 was at one point implemented as a wrapper for a pyrex module. This was removed ages ago, and trying to call `get_frequency_spectrum` with `model=Alvarez2012` would already have returned an error. Now at least the error is raised one module earlier : )

Reply:
nice catch :)